ingestion/src/metadata/ingestion/source/dashboard/superset/db_source.py

```diff
@@ -62,26 +74,32 @@ def prepare(self):
         """
         charts = self.engine.execute(FETCH_ALL_CHARTS)
         for chart in charts:
-            self.all_charts[chart["id"]] = dict(chart)
+            chart_detail = FetchChart(**chart)
+            self.all_charts[chart_detail.id] = chart_detail
+
+    def get_column_list(self, table_name: FetchChart) -> Optional[List[object]]:
+        sql_query = sql.text(FETCH_COLUMN.format(table_name=table_name.lower()))
+        col_list = self.engine.execute(sql_query)
+        return [FetchColumn(**col) for col in col_list]
 
     def get_dashboards_list(self) -> Optional[List[object]]:
```

Comment:
same thing here it should not be `object` it should be `Iterable[FetchDashboard]`